src/main/java/org/openjdk/jextract/impl/HeaderFileBuilder.java

```diff
@@ -186,7 +189,11 @@ public interface \{invokerName} {
             appendLines(STR."""
                 public static \{invokerName} \{invokerFactoryName}(MemoryLayout... layouts) {
                     FunctionDescriptor baseDesc$ = \{descriptorString(2, descriptor)};
-                    var mh$ = RuntimeHelper.downcallHandleVariadic("\{nativeName}", baseDesc$, layouts);
+                    var mh$ = Linker.nativeLinker().downcallHandle(
```

Comment:
If we keep other handy generation wrappers, we might keep this as well. It's not super trivial after all.